MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -116,7 +116,15 @@ + (IMP)swizzleOriginalSelector:(SEL)originalSelector withCustomSelector:(SEL)cus
   IMP customImp = class_getMethodImplementation(self, customSelector);
   IMP originalImp = NULL;
   BOOL methodAdded = NO;
-  
+
+  // Replace implementation in super classes
+  Class baseClass = originalClass.superclass;
```

Comment:
Can we be sure that originalClass actually responds to superclass? In practice: yes, in theory: no. So we should check that originalClass actually responds to `superclass`.

Or am I too paranoid @jaelim-ms @clpolet ?